ui/app/AppLayouts/Wallet/popups/simpleSend/SendSignModal.qml

```diff
@@ -0,0 +1,308 @@
+    //: e.g. (Send) 100 DAI to batista.eth
+    subtitle: {
+        const tokenToSend = root.isCollectible ? root.collectibleName:
+                              "%1 %2".arg(root.tokenAmount).arg(root.tokenSymbol)
+        return qsTr("%1 to %2").
+        arg(tokenToSend).
+        arg(SQUtils.Utils.elideAndFormatWalletAddress(root.recipientAddress))
+    }
+
+    // Wallet account background color to be used as gardient in the header
+    gradientColor: root.accountColor
+
+    // In case if selected token is an asset then this displays the account selected
+    fromImageSmartIdenticon.asset.name: "filled-account"
+    fromImageSmartIdenticon.asset.emoji: root.accountEmoji
+    fromImageSmartIdenticon.asset.color: root.accountColor
+    fromImageSmartIdenticon.asset.isLetterIdenticon: !!root.accountEmoji
+
+    // In case if selected token is an asset then this displays the token selected
+    toImageSource: Constants.tokenIcon(root.tokenSymbol)
+
+    // Collectible data in header in case a collectible is selected
+    collectibleMedia.backgroundColor: root.collectibleBackgroundColor
+    collectibleMedia.isMetadataValid: root.collectibleIsMetadataValid
+    collectibleMedia.mediaUrl: root.collectibleMediaUrl
+    collectibleMedia.mediaType: root.collectibleMediaType
+    collectibleMedia.fallbackImageUrl: root.collectibleFallbackImageUrl
+
+    /** In case if selected token is an collectible then
+    this displays the account selected as badge **/
+    accountSmartIdenticon.asset.name: "filled-account"
+    accountSmartIdenticon.asset.emoji: root.accountEmoji
+    accountSmartIdenticon.asset.color: root.accountColor
+    accountSmartIdenticon.asset.isLetterIdenticon: !!root.accountEmoji
+    accountSmartIdenticon.asset.isImage: root.isCollectible
+
+    //: e.g. "Send 100 DAI to recipient on <network chain name>"
+    headerMainText: {
+        const tokenToSend = root.isCollectible ? root.collectibleName:
+                               "%1 %2".arg(root.tokenAmount).arg(root.tokenSymbol)
+        return qsTr("Send %1 to %2 on %3").arg(tokenToSend)
+        .arg(SQUtils.Utils.elideAndFormatWalletAddress(root.recipientAddress)).arg(root.networkName)
+    }
+    infoTagText: qsTr("Review all details before signing")
+
+    headerIconComponent: StatusSmartIdenticon {
+        asset.name: "filled-account"
+        asset.emoji: root.accountEmoji
+        asset.color: root.accountColor
+        asset.isLetterIdenticon: !!root.accountEmoji
+        asset.bgWidth: 40
+        asset.bgHeight: 40
+
+        bridgeBadge.visible: true
+        bridgeBadge.border.width: 2
+        bridgeBadge.color: Theme.palette.darkBlue
+        bridgeBadge.image.source: Theme.svg("sign")
+    }
+
+    leftFooterContents: ObjectModel {
+        RowLayout {
+            Layout.leftMargin: 4
+            spacing: Theme.bigPadding
+            ColumnLayout {
+                spacing: 2
+                StatusBaseText {
+                    text: qsTr("Est time")
```

Comment:
Note, to align with the UI: this one has to follow the selected fee option, you don't need to do anything here, I will add this when adding transaction settings.